android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/KMPBrowserActivity.java

```diff
@@ -162,6 +163,15 @@ protected void onPause() {
   @Override
   protected void onDestroy() {
     super.onDestroy();
+    // Blank the webView and destroy completely
+    if (webView != null) {
+      webView.loadUrl("about:blank");
+      ((ViewGroup) webView.getParent()).removeView(webView);
```

Comment:
Is there ever a chance that parent could be null?

Reply:
Added a null check